pmd-javascript/src/main/resources/category/ecmascript/errorprone.xml

```diff
@@ -93,12 +93,12 @@ if (someVar != 3) {
         </example>
     </rule>
 
-    <rule name="InnaccurateNumericLiteral"
+    <rule name="InaccurateNumericLiteral"
```

Comment:
This change is actually tracked by #2367. Please remove this from this PR as changing the name would break compatibility, and needs to be handled with care. If you want to tackle the issue, open a separate PR with only those changes

Reply:
@oowekyala done.